coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/AbstractSqlLogger.scala

```diff
@@ -181,7 +181,11 @@ abstract class AbstractSqlLogger[CT, CV](val connection: Connection,
       case Some(JBoolean(b)) => b
       case _ => false
     }
-    logLine(IndexDirectiveCreatedOrUpdated, LogData.IndexDirectiveCreatedOrUpdated(convert(info.unanchored), enabled))
+    val search = directive.get("search") match {
+      case Some(JBoolean(b)) => Some(b)
+      case _ => Some(true)
```

Comment:
should this be None?

Reply:
This is kinda intentional just in case the search key isn't a boolean.